docs/devel/development.md

```diff
@@ -325,9 +325,10 @@ Three basic commands let you run unit, integration and/or e2e tests:
 
 ```sh
 cd kubernetes
-make test  # Run unit tests
+make test # Run every unit test
+make test WHAT=pkg/util/cache GOFLAGS=-v # Run tests of a package verbosely
 make test-integration  # Run integration tests, requires etcd
-go run hack/e2e.go -v --build --up --test --down  # Run e2e tests
+make test-e2e # Run e2e tests
```

Comment:
Do you have info for how to run e2e against a specific cluster using the makefile? How to run a specific test with the ginkgo args? Essentially the same information that you added for the unit test invocations, but for e2e.